apps/prairielearn/src/lib/db-types.ts

```diff
@@ -283,6 +283,13 @@ export const Lti13InstanceSchema = z.object({
 });
 export type Lti13Instance = z.infer<typeof Lti13InstanceSchema>;
 
+export const Lti13UserSchema = z.object({
+  lti13_instance_id: IdSchema,
+  sub: z.string(),
+  user_id: IdSchema.nullable(),
```

Comment:
To account for the changes from 421af14:

```suggestion
  user_id: IdSchema,
```

Reply:
Fixed in 4932db8a